### ISSUE TITLE
[Bug] - Add new languages to humanizers.

### ISSUE THREAD

**Issue by slorenzo** [EvanHahn/HumanizeDuration.js#160]
Hi.

I just followed the example provided on the docs to add a new language.

```
const shortEnglishHumanizer = humanizeDuration.humanizer({
  language: 'shortEn',
  languages: {
    shortEn: {
      y: () => 'y',
      mo: () => 'mo',
      w: () => 'w',
      d: () => 'd',
      h: () => 'h',
      m: () => 'm',
      s: () => 's',
      ms: () => 'ms',
    },
  },
});
```

After that, I have received: `1 m, 11undefined75 s` from `71750` milliseconds.

Thanks for this awesome library.

**Comment by EvanHahn** [comment]
This looks like a bug. To fix this in the short term, add a `decimal` option:

```js
const shortEnglishHumanizer = humanizeDuration.humanizer({
  language: 'shortEn',
  languages: {
    shortEn: {
      // ...
      decimal: '.'
    },
  },
});
```

I'll plan to fix this soon.

**Comment by EvanHahn** [comment]
This has been fixed in `humanize-duration@3.20.1`. Thanks for reporting!
